CODEOWNERS

```
@@ -178,6 +178,7 @@
 /doc/nrf/libraries/nrf_rpc/nrf_rpc_ipc.rst @nrfconnect/ncs-si-muffin-doc
 /doc/nrf/libraries/nrf_rpc/nrf_rpc_uart.rst @nrfconnect/ncs-terahertz-doc
 /doc/nrf/libraries/nrf_rpc/nrf_rpc_utils.rst @nrfconnect/ncs-terahertz-doc
+/doc/nrf/libraries/otdoa_al/              @nrfconnect/ncs-iot-positioning-doc @nriedel
```

We cannot have external people in CODEOWNERS so we'll need to add you manually to the PRs.

```suggestion
/doc/nrf/libraries/otdoa_al/              @nrfconnect/ncs-iot-positioning-doc
```